SqlDataCatalog-Collibra/convert-to-collibra.ps1

```diff
@@ -0,0 +1,203 @@
+        type         = [pscustomobject]@{
+            name = "Technology Asset Domain"
+        }
+    }
+
+    $export += [pscustomobject]@{
+        resourceType = "Domain";
+        identifier   = $physicalModelIdentifier;
+        type         = [pscustomobject]@{
+            name = "Physical Data Dictionary"
+        }
+    }
+
+    # Instance and database
+
+    $acceptedStatus = [pscustomobject]@{
+        name = "Accepted"
+    }
+    $instanceIdentifier = [pscustomobject]@{
+        name   = $instanceName;
+        domain = $systemsAndDatabasesIdentifier
+    }
+    $export += [pscustomobject]@{
+        resourceType = "Asset";
+        identifier   = $instanceIdentifier;
+        displayName  = $instanceName;
+        type         = [pscustomobject]@{
+            name = "Server"
+        };
+        status       = $acceptedStatus;
+    }
+
+    $databaseIdentifier = [pscustomobject]@{
+        name   = "$instanceName>$databaseName";
+        domain = $systemsAndDatabasesIdentifier
+    };
+    $export += [pscustomobject]@{
+        resourceType = "Asset";
+        identifier   = $databaseIdentifier;
+        displayName  = $databaseName;
+        type         = [pscustomobject]@{
+            name = "Database"
+        };
+        status       = $acceptedStatus;
+        relations    = [pscustomobject]@{
+            "00000000-0000-0000-0000-000000007054:SOURCE" = @($instanceIdentifier)
+        }
+    }
+
+    $columns | Group-Object schemaName | ForEach-Object {
+
+
+        # schema
+
+        $schemaName = $_.Name
+        $schemaIdentifier = [pscustomobject]@{
+            name   = "$instanceName>$databaseName>$schemaName";
+            domain = $physicalModelIdentifier
+        };
+
+        $export += [pscustomobject]@{
+            resourceType = "Asset";
+            identifier   = $schemaIdentifier
+            displayName  = $schemaName;
+            type         = [pscustomobject]@{
+                name = "Schema"
+            };
+            status       = $acceptedStatus;
+            relations    = [pscustomobject]@{
+                "00000000-0000-0000-0000-000000007005:TARGET" = @($databaseIdentifier)
+            }
+        }
+
+        $_.group | Group-Object tableName | ForEach-Object {
+            # table
+
+            $tableName = $_.Name
+            $tableIdentifier = [pscustomobject]@{
+                name   = "$instanceName>$databaseName>$schemaName>$tableName";
+                domain = $physicalModelIdentifier
+            };
+            $export += [pscustomobject]@{
+                resourceType = "Asset";
+                identifier   = $tableIdentifier;
+                displayName  = $tableName;
+                type         = [pscustomobject]@{
+                    name = "Table"
+                };
+                status       = $acceptedStatus;
+                relations    = [pscustomobject]@{
+                    "00000000-0000-0000-0000-000000007043:SOURCE" = @($schemaIdentifier)
```

Comment:
Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?